core-chart/templates/postcreatehooks/its-status.yaml

```diff
@@ -0,0 +1,25 @@
+{{- if .Values.InstallPCHs }}
+apiVersion: tenancy.kflex.kubestellar.org/v1alpha1
+kind: PostCreateHook
+metadata:
+  name: its-status
```

Comment:
The name "its-status" is not particularly clear. This is not about the status of the ITS. It is about installing the OCM status Add-On. Perhaps a name like "install-status-addon"?